src/main/java/core/basesyntax/Program.java

```diff
@@ -0,0 +1,251 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Program {
+
+    private static String readFromConsole() {
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+        String command = "";
+        System.out.println("Please, write command and press ENTER");
+        try {
+            command = bufferedReader.readLine();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return command;
+    }
+
+    private static void findCommand(String command) {
```

Comment:
```suggestion
    private static void executeCommand(String command) {
```
адже вони тут виконуються, команда вже передається в параметрах